src/jmc/compile/utils.py

```diff
@@ -248,3 +249,22 @@ def rand_java_int() -> int:
 
 def is_decorator(string: str) -> bool:
     return (len(string) > 2 and string.startswith("@"))
+
+def deep_merge(first: dict, second: dict) -> dict:
+    """
+    Recursively merges two dictionaries together.
+    (i.e. it also merges any dicts inside of them)
+
+    :param first: The dictionary to merge stuff to
+    :param second: The dictionary being merged into the first
+    :return: A new dict that is the merger of both inputs
+    """
+    print("\033[96;40m", f"first: {first}", "\033[0;0;40m")
```

Comment:
Is this supposed to be here?

Reply:
Shoot I missed one when removing print statements